WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -2,14 +2,10 @@
 
 <%= render layout: "nav" do %>
 
-  <% unless @competition.registration_requirements.blank? %>
-    <p><b><%= t('registrations.registration_requirements') %></b>
-    </p>
-    <div>
-      <%=md @competition.registration_requirements %>
-    </div>
-    <hr/>
-  <% end %>
+  <p><b><%= t('registrations.registration_requirements') %></b>
```

Comment:
It's not clear to me why it's safe to remove the `unless` here. Will this do something weird for upcoming competitions that haven't specified registration requirements?

Reply:
I remember now why I changed this: `@competition.registration_requirements` can be blank with the changes I made, but that doesn't mean that the _registration_requirements partial doesn't have to be rendered, so I think this is as it should be, i.e. without the `unless`.

Reply:
Sounds good. If something weird shows up, we can always fix it =)